validator/lib/validateTileset.js

```diff
@@ -36,19 +36,19 @@ function validateTileset(tileset, tilesetDirectory) {
 
 function validateTopLevel(tileset) {
     if (!defined(tileset.geometricError)) {
-        return 'Tileset must declare its geometricError as a top-level property.';
+        return errorMessage('Tileset must declare its geometricError as a top-level property.', undefined);
```

Comment:
These at the top level don't need to go through `errorMessage`. Just returning the message is fine.